.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,156 @@
+name: Bug report
+description: Broken or unintended behavior with one of our libraries.
+title: '[Bug] '
+labels: ["untriaged", "needs attention"]
+body:
+- type: markdown
+  attributes:
+    value: |
+      ## Before submitting your issue
+      Please make sure that your question or issue is not already covered in existing issues, [MSAL docs](https://learn.microsoft.com/en-us/entra/msal/dotnet/), or [samples](https://learn.microsoft.com/en-us/azure/active-directory/develop/sample-v2-code?tabs=apptype).
+
+      **Logs and network traces**
+      Without logs or traces, it is unlikely that the team can investigate your issue. Capturing logs and network traces is described in our [Docs](https://learn.microsoft.com/azure/active-directory/develop/msal-logging-dotnet).
+
+- type: markdown
+  attributes:
+    value: |
+      ## Issue details   
+    
+- type: input
+  attributes:
+    label: Library version used
+    description: "Please enter the version of the library where you ran into the issue (e.g. 4.54.0)."
+  validations:
+    required: true
+    
+- type: textarea
+  attributes:
+    label: .NET version
+    description: "Please enter the .NET platform version your application was developed in."
+  validations:
+    required: true
+
+- type: dropdown
+  attributes:
+    label: MSAL client type
+    description: "Are you using PublicClientApplication,  ConfidentialClientApplication or ManagedIdentityApplication?"
+    multiple: true
+    options: 
+      - "Public"
+      - "Confidential"
```

Comment:
Can you control the next 2 drop downs based on what is selected here? 

PCA > show app type as "desktop with UI", "console app", "mobile" and the auth flows 
CCA > show only "web app", "web api", "service 2 service". No need for auth flows, they are implied.
MI > maybe ask for the host used?

Reply:
@bgavrilMS no easy way to show or hide based on selection. if there are any pointers on how to do it then I can give it a try

Reply:
Will add a task for this and see if there is a way to do this